scripts/add_indexes.py

```
@@ -0,0 +1,97 @@
+        """
+    ]
+
+    with engine.connect() as conn:
+        with conn.begin():
+            # Postgres GIN indexes require the pg_trgm extension for some advanced text operations,
+            # though to_tsvector doesn't strictly need it, it's good to have.
+            add_pg_trgm_extension = os.getenv("ADD_PG_TRGM_EXTENSION", "true").lower() in ("1", "true", "yes", "on")
+            if add_pg_trgm_extension:
+                try:
+                    conn.execute(text('CREATE EXTENSION IF NOT EXISTS "pg_trgm";'))
+                except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)